CHANGELOG.md

```diff
@@ -1,3 +1,52 @@
+# 107.0.0
+
+In this major release we deleted the deprecated themer classes: `MDCAlertScheme` and `MDCAlertControllerThemer`.
+
+## Breaking changes
+
+Deleted `MDCAlertScheme` and `MDCAlertControllerThemer` class, please use `MDCAlertController (MaterialTheming)` extension instead.
```

Comment:
`MDCAlertScheme` and `MDCAlertControllerThemer` have been deleted. Please use the `MDCAlertController (MaterialTheming)` extension instead.